versions.props

```diff
@@ -1,5 +1,5 @@
 ch.qos.logback:logback-classic = 1.1.2
-com.fasterxml.jackson.*:* = 2.9.9
```

Comment:
@schlosna we've actually stopped auto-upgrading some jars in tracing-java, to avoid imposing upgrades on people when they're not really necessary.

https://github.com/palantir/tracing-java/blob/develop/versions.props

I think we'd probably want applications (and probably witchcraft) to always be upgrading everything, but I think there might be an argument to have libraries bump stuff a bit less aggressively to make life easier for consumers?

Reply:
Hey @iamdanfox, that makes sense. Excavator had bumped jackson for tritium this morning. 

I think we actually do want to take the Jackson 2.9.9 bump ASAP given the [Afterburner issues we've fixed upstream](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.9.9#afterburner) in this release.

Reply:
Speaking of the bots, https://github.com/palantir/tritium/pull/266 interestingly bumped just the `versions.lock` for `com.fasterxml.jackson.datatype:jackson-datatype-guava` to `2.9.9`